spec/system/admin/configuration/terms_of_service_files_spec.rb

```diff
@@ -12,35 +12,41 @@
 
     it "can be reached via Configuration" do
       visit spree.edit_admin_general_settings_path
-      click_link "Terms of Service"
-
-      expect(page).to have_content "No terms of services have been uploaded yet."
-      expect(page).to have_content "your old Terms of service"
-      expect(page).to have_link "Terms of service", href: "/Terms-of-service.pdf"
+      click_link I18n.t("admin.terms_of_service_files.show.terms_of_service")
```

Comment:
I'd be happy if @mkllnk nor @filipefurtad0 could review this change as well. Not sure we want to use I18n and i18n keys in the specs file, but maybe I'm wrong.

Reply:
Thanks for the heads up @jibees.
Best not to test I18n and i18n keys in system specs - as per [this review](https://github.com/openfoodfoundation/openfoodnetwork/pull/9350#pullrequestreview-1030953353) and [this one](https://github.com/openfoodfoundation/openfoodnetwork/pull/8319#discussion_r725804103) - and simply assert that Capybara finds the clickable corresponding string on the page:

```suggestion
      click_link "Terms of Service"
```

I agree this is confusing @aintluks as our specs still test I18n in several places. Something to improve on our side :+1: